src/main/kotlin/blackjack/BlackjackGame.kt

```diff
@@ -0,0 +1,68 @@
+package blackjack
+
+class BlackjackGame(gamers: List<UserRole>, deck: Deck) {
+
+    private val _gamers: MutableList<UserRole> = gamers.toMutableList()
+    private val cards = deck.cards.toMutableList()
+
+    init {
+        validateMinPlayer(_gamers.size)
+
+        repeat(BASIC_RULE_COUNT) {
+            _gamers.forEach {
+                it.draw(cards.removeAt(TOP_CARD))
+            }
+        }
+    }
+
+    val gamers: List<UserRole>
+        get() = _gamers.toList()
+
+
+    fun play(): List<UserRole> {
+        val players = _gamers.filter { !it.isDealer() }
+        val results = mutableListOf<UserRole>()
+        for (player in players) {
+            results.add(playPlayerTurn(player))
+        }
+
+        results.add(playDealerTurn())
+        return results.toList()
+    }
```

Comment:
코틀린 확장 함수를 활용하면 아래처럼 작성해볼 수 있겠네요 :)
```suggestion
    fun play(): List<UserRole> {
        return _gamers.asSequence()
            .filter { !it.isDealer() }
            .map { playPlayerTurn(it) }
            .plus(playDealerTurn())
            .toList()
    }
```

Reply:
오.. plus 라는 행위가 시퀀스를 통해서 이루어지네요!! 좋은 기능 하나 배워갑니다!! 감사합니다~